qt/scientific_interfaces/test/ISISReflectometry/Reduction/ParseReflectometryStringsTest.h

```diff
@@ -140,6 +140,17 @@ class ParseReflectometryStringsTest : public CxxTest::TestSuite {
     TS_ASSERT(!result.is_initialized());
   }
 
+  void testParseScaleFactorRejectsZero() {
+    auto result = parseScaleFactor("0.0");
+    TS_ASSERT(!result.is_initialized());
+  }
+
+  void testParseScaleFactorNegative() {
+    auto result = parseScaleFactor("-1.0");
+    TS_ASSERT(result.get().is_initialized());
+    TS_ASSERT_EQUALS(result.get().get(), -1.0);
```

Comment:
Similar confusion here, see lines 109-111. I think your implementation is correct, however it looks like `TS_ASSERT_EQUALS(result.get().get(), "1-3");` may need `1-3` without the quotation marks?

Reply:
I think this line doesn't attempt to parse the run numbers (1-3) into a list of int's since we have something in the Mantid Kernel which does this. 
It's a good spot though as initially I thought someone had mistyped 1e-3 too